modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/TableViewBehaviorBase.java

```diff
@@ -135,6 +135,8 @@ public TableViewBehaviorBase(C control) {
         // create a map for TableView(Base)-specific mappings
         tableViewInputMap = createInputMap();
 
+        boolean rtl = isRTL();
+
```

Comment:
node orientation can change during the lifetime of a node ...

Reply:
Addressed this. Please review latest changes.